src/sphinx_book_theme/nodes.py

```diff
@@ -0,0 +1,36 @@
+from docutils import nodes
+from sphinx.application import Sphinx
+from typing import Any, cast
+
+
+class SideNoteNode(nodes.Element):
+    """A node that will not be rendered."""
```

Comment:
This doesn't sound like the correct docstring

Reply:
Oops.

Reply:
done in [a87245f](https://github.com/executablebooks/sphinx-book-theme/pull/546/commits/a87245f0706bbf8c070a78a6329783289cc7f1e1)